### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by TanOfAllCodes**
include iperf3, tcpdump, traceroute

### CODE REVIEW

**Comment by herlesupreeth** [comment]
Hey @TanOfAllCodes , Thanks for the PR. I am bit against adding 

1. tcpdump - because normally one would take the trace on the hostmachine running the entire docker setup and wont be needing trace on per container level. Please let me know if you think otherwise
2. iperf3 - is there any specific reason to use iperf3 vs iper (iperf is already bundled with the image)

File: .env [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #TZ=Europe/Berlin
 
 MCC=001
-MNC=01
+MNC=44
```
Comment (herlesupreeth) [inline .env:5]: Please revert this as I plan to keep the default MCC MNC as test network PLMN of 001 01

**Comment by TanOfAllCodes** [comment]
@herlesupreeth thanks for the suggestion, yes I reverted the local .env as I agree it is the standard. 

tcpdump is increasingly useful when the core deployment starts getting more complex - having multiple machines with different docker containers, and also running it for specific containers. especially for the UPF, which is built from the base image anyway and that also has a use case.

iperf3 and iperf also provide a few different features. iperf is legacy and iperf3 has more support for modern features ex: ipv6 and integrates well with structuring output (so log files as better)

**Comment by herlesupreeth** [comment]
Thank you

**Pull request merged**